src/executors/ruby.yml

```diff
@@ -0,0 +1,14 @@
+description: Box just Ruby
+parameters:
+  ruby_version:
+    type: string
+    default: '2.5.5'
+docker:
+  - image: circleci/ruby:<< parameters.ruby_version >>-node-browsers-legacy
```

Comment:
Do you think the `node-browsers-legacy` is still useful for this executor?
🤔Maybe for engines that don't involve solr/fedora like QA and ActiveEncode.

Reply:
Yeah, I left it because I plan to use this for QA.